src/Parser.js

```diff
@@ -839,6 +839,24 @@ Parser.prototype.parseSymbol = function() {
         return new ParseFuncOrArgument(
             new ParseNode("textord", nucleus.text, this.mode, nucleus),
             false, nucleus);
+    } else if (/^\\verb[^a-zA-Z]/.test(nucleus.text)) {
+        this.consume();
+        var arg = nucleus.text.slice(5);
+        var star = (arg.charAt(0) === "*");
+        if (star) {
+            arg = arg.slice(1);
+        }
+        // Lexer's tokenRegex is constructed to always have matching
+        // first/last characters.
+        if (arg.length < 2 || arg.charAt(0) !== arg.slice(-1)) {
+            throw new ParseError("\\verb failure to parse");
```

Comment:
What input would trigger this? Actually it might make sense to document error conditions in [`test/errors-spec.js`](https://github.com/Khan/KaTeX/blob/master/test/errors-spec.js).